docs/guides/use-connectors-in-hybrid-mode.md

```diff
@@ -0,0 +1,163 @@
+---
+id: use-connectors-in-hybrid-mode
+title: Use Connectors in hybrid mode
+description: "Learn how to run Connectors in hybrid mode."
+---
+
+:::note
+Hybrid mode is supported as of the Connectors `0.23.0` release.
+:::
+
+**Hybrid mode** is where you can run a Self-Managed Connectors runtime instance attached to a Camunda SaaS cluster or another Self-Managed cluster that has another instance of the Connectors runtime attached.
+
+To name few use-cases where this approach might be useful:
+
+- When you deal with services that must be isolated within private network and must never be exposed to the public internet.
+- Infrastructure amendments need to be applied to the Connectors runtime, such as SSL certificates, mounted volumes, etc.
+- Code modifications applied to Connectors runtime, or specific connector logic.
+
+## How it works
+
+Every Connector has its ID (type definition), and name. Every Connector element template has a hidden property that
+defines which Connector is to be used to execute with a given template.
+
+For example, see a relation between [Kafka element template](https://github.com/camunda/connectors/blob/main/connectors/kafka/element-templates/kafka-inbound-connector.json#L39)
+and [Kafka Connector](https://github.com/camunda/connectors/blob/main/connectors/kafka/src/main/java/io/camunda/connector/kafka/inbound/KafkaExecutable.java#L20).
+
+To make hybrid Connector runtime work correctly, it is required to override Connector type.
```

Comment:
Do you mean "For the hybrid Connectors runtime to work properly, you must override the Connector type."?

Also note, I have an open issue regarding "Connector runtime" or "Connectors" runtime because we use this inconsistently. https://github.com/camunda/developer-experience/issues/113

Reply:
That's correct. Fixed it.